test/addressbook.json

```diff
@@ -8231,6 +8231,9 @@
         "stateMutability": "view",
         "type": "function"
       }
-    ]
+    ],
+    "events_to_watch": [],
+    "proof_format": "POS",
```

Comment:
We're actually still using "POW" for all contracts. Changing this might introduce a new behavior.